cypress/e2e/build-your-plate.cy.ts

```diff
@@ -1,6 +1,257 @@
+import terminalLog from "../support/terminal-log";
+import foodList from "../fixtures/build-your-plate.json";
+
 describe("Build Your Plate", () => {
-  it("gives correct score with 5 sweets");
-  it("gives correct score with 5 meats");
-  it("gives correct score with 5 mixed items");
-  it("allows user to swap food items before scoring");
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/Games/BuildYourPlate");
+  });
+
+  it("check a11y buuld-your-plate page", () => {
+    cy.injectAxe();
+    cy.checkA11y(null, null, terminalLog, true);
+  });
+
+  it("can navigate to the game page", () => {
+    cy.visit("http://localhost:3000/");
+    cy.get(".react-multi-carousel-list [data-testid='Games']").click();
+    cy.get("[data-testid='menu-styling'] [data-testid='Build your plate']").click();
+    cy.url().should("eq", "http://localhost:3000/Games/BuildYourPlate");
+  });
+
+  it("gives correct score with 5 sweets", () => {
+    cy.get("[data-testid='game-begin']").click();
+    cy.contains("button", "Play").click();
+    cy.contains("button", "Score my plate").should("be.disabled");
+    cy.contains("0 / 5");
+    cy.get("button [alt='Sweets']", { timeout: 10000 }).click();
+
+    Object.keys(foodList.sweets).forEach((item) => {
+      cy.get("body").then((body) => {
+        if (body.find(`button [alt='${item}']`).length) {
+          cy.get(`button [alt='${item}']`).click();
+        }
+      });
+    });
+
+    cy.get("[alt='Tick']").should("have.length", 5);
+    cy.contains("button", "Score my plate").should("be.enabled");
+    cy.contains("5 / 5");
+    cy.contains("button", "Score my plate").click();
+    cy.get("[alt='plate']");
+    cy.contains("button", "Score my plate").click();
+    cy.contains("Score: 10 out of 50");
+    cy.contains("Could be better!");
+    cy.contains("button", "Play again").click();
+  });
+
+  it("gives correct score with 5 meats", () => {
```

Comment:
Unsure if testing 5 meats after 5 sweets is necessary as not testing different functionality.
Might be more worthwhile to have a test for 5 of the same category and 5 of different mixed categories.